src/components/Layout/Section/RSVDetails.tsx

```diff
@@ -22,121 +22,133 @@ const stats = {
 };
 
 type Props = {
-  radschnellweg: {
-    ref: string;
-    name: string;
-    website: string;
-    status: string;
-    to: string;
-    from: string;
-    finished?: string;
-    costs?: number;
-    coordinates: Array<Array<number>>;
+  meta: {
+    finished: string;
+    state: string;
+    cost: string;
+    general: {
+      name: string;
+      description: string;
+      to: string;
+      from: string;
+      ref: string;
+    };
+    references: {
+      website: string;
+    };
+  };
+  geometries: {
+    nodes: Array<{
+      id: string;
+      type: 'Feature';
+      geometry: GeoJSON.Geometry;
+      properties: {
+        id_rsv: string;
+        id_segment: string;
+        length: string;
+        planning_phase: string;
+        status: string;
+        variants: string;
+        detail_level: string;
+      };
+    }>;
   };
 };
 
-const summary =
-  'Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At vero eos et accusam et justo duo dolores et ea rebum. Stet clita kasd gubergren, no sea takimata sanctus est Lorem ipsum dolor sit amet. Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At vero eos et accusam et justo duo dolores et ea rebum. Stet clita kasd gubergren, no sea takimata sanctus est Lorem ipsum dolor sit amet.';
+const colors = [
```

Comment:
Empfehlung: In eine `lineWidth.const.ts` extrahieren um diese Datei schlanker zu halten.